src/utils/hashUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { createHash } from 'crypto'
+
+export enum HashDigest {
+  HEX = 'hex',
+  BASE64 = 'base64',
+}
+
+export enum HashAlgorithm {
+  SHA256 = 'sha256',
+  SHA512 = 'sha512',
+}
+
+export function generateHash(data: string, algorythm: HashAlgorithm, digest: HashDigest) {
+  return createHash(algorythm).update(data).digest(digest)
```

Comment:
what do you think about using https://nodejs.org/api/crypto.html#cryptohashalgorithm-data-outputencoding?

Reply:
Yeap, sounds good to stick to the proposed order of params, and using similar naming. 
Will do some changes.

Reply:
Using the `hash` approach won't work in node v18:

```
crypto.hash(algorithm, data[, outputEncoding])[#](https://nodejs.org/api/crypto.html#cryptohashalgorithm-data-outputencoding)
Added in: v21.7.0, v20.12.0
```

I just changed the order and naming (encoding instead of digest) to keep it similar to `crypto.hash`

https://github.com/lokalise/node-core/pull/135/commits/398cd9224d8640d3606cd877bbd5b474d4a24bb2

Reply:
We don't need to support Node 18, as we are actively phasing it out from everywhere, but I don't have a strong opinion on this, we can keep the createHash approach, especially since `hash` is still at Release Candidate stage.